jspdf.js

```diff
@@ -777,11 +777,15 @@ var jsPDF = (function(global) {
 			if (fontName !== undefined){
 				fontName = fontName.toLowerCase();
 			}
+		   /**
+			* only a few font have metrics, see standard_fonts_metrics.js
+			*
+			*/
 			switch(fontName){
+			case 'calibri':
```

Comment:
Why do you add calibri to the switch operation? this should not be made here. maybe in from_html, but not in  standard pdf generation